Go 1.15rc1 error format change in ParseDuration

**Describe the bug**
```
Testing    in: /builddir/build/BUILD/go-toml-1.8.0/_build/src
         PATH: /builddir/build/BUILD/go-toml-1.8.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/go-toml-1.8.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/pelletier/go-toml/version=1.8.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/pelletier/go-toml
github.com/pelletier/go-toml
{Hello 3.5 {One} [{Two} {Three}]}
--- FAIL: TestUnmarshalBadDuration (0.00s)
    marshal_test.go:2157: unexpected error: (1, 1): Can't convert 1z(string) to time.Duration. time: unknown unit "z" in duration "1z"
FAIL
```

**To Reproduce**
Running go test.

**Expected behavior**
Tests pass.

**Versions**
 - go-toml: version 1.8.0
 - go: version Go 1.15 rc 1
 - operating system: Fedora Linux Rawhide

**Additional context**
The error message was modified to add quote around 1z and z, so it is not recognized by
https://github.com/pelletier/go-toml/blob/34de94e6a887dcd421ab7586d76d26ee5ab3d96a/marshal_test.go#L2199

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -13,9 +13,9 @@ stages:
       vmImage: ubuntu-latest
     steps:
     - task: GoTool@0
-      displayName: "Install Go 1.14"
+      displayName: "Install Go 1.15"
       inputs:
-        version: "1.14"
+        version: "1.15"
     - script: echo "##vso[task.setvariable variable=PATH]${PATH}:/home/vsts/go/bin/"
     - script: mkdir -p ${HOME}/go/src/github.com/pelletier/go-toml
     - script: cp -R . ${HOME}/go/src/github.com/pelletier/go-toml
@@ -36,9 +36,9 @@ stages:
       vmImage: ubuntu-latest
     steps:
     - task: GoTool@0
-      displayName: "Install Go 1.14"
+      displayName: "Install Go 1.15"
       inputs:
-        version: "1.14"
+        version: "1.15"
     - task: Go@0
       displayName: "go fmt ./..."
       inputs:
@@ -51,9 +51,9 @@ stages:
       vmImage: ubuntu-latest
     steps:
     - task: GoTool@0
-      displayName: "Install Go 1.14"
+      displayName: "Install Go 1.15"
       inputs:
-        version: "1.14"
+        version: "1.15"
     - task: Go@0
       displayName: "Generate coverage"
       inputs:
@@ -71,9 +71,9 @@ stages:
       vmImage: ubuntu-latest
     steps:
     - task: GoTool@0
-      displayName: "Install Go 1.14"
+      displayName: "Install Go 1.15"
       inputs:
-        version: "1.14"
+        version: "1.15"
     - script: echo "##vso[task.setvariable variable=PATH]${PATH}:/home/vsts/go/bin/"
     - task: Bash@3
       inputs:
@@ -86,9 +86,9 @@ stages:
       vmImage: ubuntu-latest
     steps:
     - task: GoTool@0
-      displayName: "Install Go 1.14"
+      displayName: "Install Go 1.15"
       inputs:
-        version: "1.14"
+        version: "1.15"
     - script: echo "##vso[task.setvariable variable=PATH]${PATH}:/home/vsts/go/bin/"
     - script: mkdir -p ${HOME}/go/src/github.com/pelletier/go-toml
     - script: cp -R . ${HOME}/go/src/github.com/pelletier/go-toml
@@ -102,6 +102,15 @@ stages:
     displayName: "unit tests"
     strategy:
       matrix:
+        linux 1.15:
+          goVersion: '1.15'
+          imageName: 'ubuntu-latest'
+        mac 1.15:
+          goVersion: '1.15'
+          imageName: 'macOS-latest'
+        windows 1.15:
+          goVersion: '1.15'
+          imageName: 'windows-latest'
         linux 1.14:
           goVersion: '1.14'
           imageName: 'ubuntu-latest'
@@ -110,15 +119,6 @@ stages:
           imageName: 'macOS-latest'
         windows 1.14:
           goVersion: '1.14'
-          imageName: 'windows-latest'
-        linux 1.13:
-          goVersion: '1.13'
-          imageName: 'ubuntu-latest'
-        mac 1.13:
-          goVersion: '1.13'
-          imageName: 'macOS-latest'
-        windows 1.13:
-          goVersion: '1.13'
           imageName: 'windows-latest'
     pool:
       vmImage: $(imageName)
@@ -155,7 +155,7 @@ stages:
     - task: GoTool@0
       displayName: "Install Go"
       inputs:
-        version: 1.14
+        version: 1.15
     - task: Bash@3
       inputs:
         targetType: inline

--- a/marshal_test.go
+++ b/marshal_test.go
@@ -2194,10 +2194,7 @@ func TestUnmarshalBadDuration(t *testing.T) {
 	result := testBadDuration{}
 	err := NewDecoder(buf).Decode(&result)
 	if err == nil {
-		t.Fatal()
-	}
-	if err.Error() != "(1, 1): Can't convert 1z(string) to time.Duration. time: unknown unit z in duration 1z" {
-		t.Fatalf("unexpected error: %s", err)
+		t.Fatal("expected bad duration error")
 	}
 }
 

